src/components/organisms/NewOrderBookWidget/NewOrderBookWidget.tsx

```diff
@@ -433,7 +435,7 @@ export const NewOrderBookWidget = ({
     };
 
     return (
-        <div className='flex h-full w-full flex-col justify-start gap-y-1 overflow-hidden border-white-10 laptop:flex-col-reverse laptop:gap-y-0 laptop:rounded-b-xl laptop:border laptop:bg-cardBackground/60 laptop:shadow-tab'>
```

Comment:
I had to remove this overflow-hidden because the InfoTooltip (in Itayose) was getting partially blocked by the edges of the container. I made sure to check that it did not break anything